source/FilterableLogMonitor.js

```diff
@@ -102,20 +113,26 @@ export default class FilterableLogMonitor extends Component {
           color: theme.base07,
           fontSize: 14,
           minWidth: 200
-        }}
-      >
+        }}>
         <ActionFilter
           actionFilterText={actionFilterText}
           dispatch={dispatch}
           theme={theme}
         />
         <div
+          ref={(node) => { this.wrapperNode = node }}
           style={{
             flex: '1',
             overflowY: 'scroll'
           }}
-        >
-          {filterableStates}
+          onScroll={() => {
+            if (this.wrapperNode.scrollTop + this.wrapperNode.offsetHeight >= this.containerNode.offsetHeight) {
+              this.isAutoScrollEnabled = true
+              return
+            }
+            this.isAutoScrollEnabled = false
+          }}>
+          <div ref={(node) => { this.containerNode = node }} >{filterableStates}</div>
```

Comment:
nit: The 2 `ref` setters and the `onScroll` handler would be better as class methods. This way React won't have to unset and re-set the `ref` each render.